pkg/retrieval/retrieval.go

```diff
@@ -248,22 +252,12 @@ func (s *Service) RetrieveChunk(ctx context.Context, addr swarm.Address, origin
 	return v.(swarm.Chunk), nil
 }
 
-func (s *Service) retrieveChunk(ctx context.Context, addr swarm.Address, sp *skippeers.List, originated bool) (chunk swarm.Chunk, peer swarm.Address, requested bool, err error) {
+func (s *Service) retrieveChunk(ctx context.Context, addr swarm.Address, sp *skippeers.List, origin bool) (chunk swarm.Chunk, peer swarm.Address, requested bool, err error) {
```

Comment:
it's subjective but it's easier for me when boolean vars/funcs have an verb prefix (isOrigin, allowUpstream)

Reply:
fixed